app/services/embeddable_content/images/img_tag_attributes.rb

```diff
@@ -25,12 +25,12 @@ def to_h
 
       def src
         case target
-        when :schoology      then s3_url(with_extension: true)
-        when :cc             then s3_url(with_extension: true)
+        when :schoology      then s3_url
+        when :cc             then s3_url
```

Comment:
while i know its not necessary, any reason not to be explicit here with `s3_url(with_extension: false)`

Reply:
good call @dylandechant i fixed it